.github/workflows/tests.yaml

```diff
@@ -98,6 +98,18 @@ jobs:
                      --door-lock-app ./out/linux-x64-door-lock-${BUILD_VARIANT}/chip-door-lock-app \
                      --tv-app ./out/linux-x64-tv-app-${BUILD_VARIANT}/chip-tv-app \
                   "
+            - name: Run OTA example
+              timeout-minutes: 30
+              run: |
+                  mkdir -p /tmp/ota/
```

Comment:
`/tmp/ota` is already created in ota_test.sh. I'm assuming this is necessary here as well since we log to this path in the GitHub workflow?

Reply:
Yes that's correct. Needed in .yaml to upload logs during the workflow, and needed in ota_test.sh to create the folder in /tmp/ota locally.